### PR TITLE
Fixes #1926 - content for /contact

### LINKED ISSUE #1926: add content to contact page
- [x] add content to contact page https://github.com/webcompat/webcompat.com/issues/1284
- [x] add active state to `contact` in footer

### PR DESCRIPTION
**Pull request by zoepage**
#1926 
- [x] add content
- [x] make footer link active

### CODE REVIEW

File: webcompat/templates/contact.html [hunk 1]
```diff
@@ -2,12 +2,65 @@
...
+		    		</a>
+		    	</li> 
+		    	<li>
+		    		Join our
+		    		<a href="https://webcompat.slack.com/" title="Join our Slack">
+		    			Slack
+		    		</a>
+		    	</li>
+		    </ul>
+		    <!-- <h2 class="headline-2">Ways to Contribute</h2>
```
Comment (miketaylr) [inline webcompat/templates/contact.html]: Should this commented out stuff just be deleted?

File: webcompat/templates/contact.html [hunk 2]
```diff
@@ -2,12 +2,65 @@
...
+		    	<li>
+		    		Join us on our 
+		    		<a href="https://lists.mozilla.org/listinfo/compatibility" title="Join us on our mailing list">
+		    			mailing list
+		    		</a>
+		    	</li> 
+		    	<li>
+		    		Join our
+		    		<a href="https://webcompat.slack.com/" title="Join our Slack">
+		    			Slack
```
Comment (miketaylr) [inline webcompat/templates/contact.html]: 😮

**Comment by miketaylr** [comment]
> This branch has conflicts that must be resolved

I might have merged your PRs in the wrong order... some conflicts. Sorry about that.

File: webcompat/templates/contact.html (hunk 2 above)
Comment (zoepage) [reply webcompat/templates/contact.html]: not good?

**Comment by zoepage** [comment]
@miketaylr merge conflict is fixed as well as the comments and the typos.

File: webcompat/templates/contact.html (hunk 2 above)
Comment (miketaylr) [reply webcompat/templates/contact.html]: Heh no, we've chatted about this before. All good.

**Comment by miketaylr** [comment]
Thanks @zoepage

**Pull request merged**